lib/kafkajs/_consumer.js

```diff
@@ -993,6 +1002,7 @@ class Consumer {
 
     const rdKafkaConfig = this.#config();
     this.#state = ConsumerState.CONNECTING;
+    rdKafkaConfig["js.queue_non_empty_cb"] = this.#queueNonEmptyCb.bind(this);
```

Comment:
This can be similar to other JS callback names, even in case we introduce the same configuration property name in librdkafka there will be a C++ function that calls this JS callback so it needs to be wrapped properly in any case.
```suggestion
    rdKafkaConfig.queue_non_empty_cb = this.#queueNonEmptyCb.bind(this);
```